sw/airborne/modules/air_data/air_data.c

```diff
@@ -173,6 +173,9 @@ static void send_baro_raw(struct transport_tx *trans, struct link_device *dev)
 
 static void send_air_data(struct transport_tx *trans, struct link_device *dev)
 {
+#ifdef SITL
+  air_data.airspeed = stateGetAirspeed_f();
```

Comment:
you should use `USE_NPS_AIRSPEED` in your airframe file instead

Reply:
This define is only used in fixedwing, but by adding the AbiSendMsgAIRSPEED line to the rotorcraft code in case of USE_AIRSPEED, all these ifs are no longer needed